src/interfaces/addresses-interface.ts

```diff
@@ -4,6 +4,7 @@
  * https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/address.json
  */
 export interface AddressIF {
+[x: string]: any
```

Comment:
Move this last, and indent it.

(It's a cheat to allow undeclared properties in this interface/type. Undesirable but we're not going to fix this right now.)